src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/config/DomainEventUrlConfig.kt

```diff
@@ -31,6 +31,7 @@ class DomainEventUrlConfig {
       DomainEventType.APPROVED_PREMISES_PLACEMENT_APPLICATION_ALLOCATED -> cas1["placement-application-allocated-event-detail"]
       DomainEventType.APPROVED_PREMISES_MATCH_REQUEST_WITHDRAWN -> cas1["match-request-withdrawn-event-detail"]
       DomainEventType.APPROVED_PREMISES_REQUEST_FOR_PLACEMENT_CREATED -> cas1["request-for-placement-created-event-detail"]
+      DomainEventType.APPROVED_PREMISES_ASSESSMENT_INFO_REQUEST -> cas1["assessment-info-request-detail"]
```

Comment:
I think the form we're using for our domain events is to put the action in the past tense (e.g. `application.submitted`, `booking.made`) so I think we probably ought to be sticking with that form for these new domain events throughout? :

- `assessment.info-requested`
- `assessment.info-received`

does it sound sensible to stay consistent?